db/institutions.yml

```diff
@@ -441,6 +482,9 @@ lehighu:
     - lehigh
   city: Bethlehem
   country: USA
+  departments:
+    dop:
```

Comment:
physics instead of dop?  You resolved the conversation but didn't make the change....

Reply:
@SaniHarouna-Mayer pls could we work to get this merged?  One comment left

Reply:
pushed the change - sorry for the delay